app/layout.tsx

```diff
@@ -24,23 +24,25 @@ const Layout: React.FC<Props> = ({
   );
 
   return (
-    <html lang="en">
-      <body>
-        <WagmiProvider>
-          <div className={rootClassName}>
-            <div className="max-w-screen-xl mx-auto">
-              <Head>
-                <title>{title}</title>
-                <meta name="description" content={`${description}`} />
-                <link rel="icon" href="/favicon.ico" />
-              </Head>
-              <Nav />
-              <main className="">{children}</main>
+    <html lang="en" suppressHydrationWarning>
+      <Providers>
+        <body className="transition-colors bg-white dark:bg-black">
+          <WagmiProvider>
```

Comment:
Can we move the `WagmiProvider` to the `Providers.tsx` file ?

Reply:
Thank you @chin-flags . I have combined `WagmiProvider` to the `Providers.tsx` file.